lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss

```diff
@@ -157,4 +160,23 @@
     }
   }
 
-}
\ No newline at end of file
+}
+
+
+
+// @TODO: move to proper locations
+// Temporary additions to work through fix for upload button issues.
+input[type="file"] {
```

Comment:
This needs to be nested within `.reportback` (or some other V2-specific class). Otherwise it will conflict with existing reportback implementation.